src/vorta/views/partials/treemodel.py

```diff
@@ -323,14 +323,12 @@ class DisplayMode(enum.Enum):
         #: simple list of items
         FLAT = enum.auto()
 
-    def __init__(self, parent=None):
+    def __init__(self, mode, parent=None):
```

Comment:
```suggestion
    def __init__(self, mode: 'FileTreeModel.DisplayMode', *, parent=None):
```

Reply:
What was the star for here? I added it and got errors in tests, so I kept the type annotation suggestion

Reply:
It forces to use `parent` as a kw arg. I like that for QObjects since it will increase forward compatibility.